powerauth-push-docs/sql/mysql/create_push_server_schema.sql

```diff
@@ -12,14 +12,14 @@ CREATE TABLE `push_app_credential` (
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 
 CREATE TABLE `push_device` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
   `activation_id` varchar(37) DEFAULT NULL,
   `user_id` varchar(255) DEFAULT NULL,
-  `app_id` int(11) DEFAULT NULL,
+  `app_id` bigint(20) DEFAULT NULL,
   `platform` varchar(20) DEFAULT NULL,
   `push_token` varchar(255) DEFAULT NULL,
-  `last_registered` TIMESTAMP NULL,
-  `is_active` int(11) DEFAULT NULL,
+  `timestamp_created` TIMESTAMP NULL,
```

Comment:
This is not correct - the timestamp represents "last registration". The mobile device can register multiple times and we are changing this timestamp after each re-registration.